src/main/java/org/folio/service/uc/export/TitleExportModel.java

```diff
@@ -0,0 +1,46 @@
+package org.folio.service.uc.export;
+
+import com.opencsv.bean.CsvBindByName;
+import com.opencsv.bean.CsvBindByPosition;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder(toBuilder = true)
```

Comment:
What is the reason behind using `SuperBuilder`?
Is this class going to be extended?

Reply:
@dmtkachenko fixed